libethereum/InstanceMonitor.cpp

```diff
@@ -46,23 +46,20 @@ void InstanceMonitor::initRotationParams( uint64_t _finishTimestamp ) {
     std::ofstream rotationInfoFile( m_rotationInfoFilePath.string() );
     rotationInfoFile << rotationJson;
 
-    m_finishTimestamp = _finishTimestamp;
-    LOG( m_logger ) << "Set rotation time to " << m_finishTimestamp;
+    LOG( m_logger ) << "Set rotation time to " << _finishTimestamp;
 }
 
 bool InstanceMonitor::isTimeToRotate( uint64_t _finishTimestamp ) {
```

Comment:
Better to rename the parameter to `currentTimestamp`/`blockTimestamp` or sth like this